x-pack/plugins/maps/public/components/widget_overlay/layer_control/view.js

```diff
@@ -19,29 +19,7 @@ import { LayerTOC } from './layer_toc';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
 
-export function LayerControl({ isReadOnly, isLayerTOCOpen, showAddLayerWizard, closeLayerTOC, openLayerTOC }) {
-  if (!isLayerTOCOpen) {
-    return (
-      <EuiToolTip
-        delay="long"
-        content={i18n.translate('xpack.maps.layerControl.openLayerTOCButtonAriaLabel', {
-          defaultMessage: 'Expand layers panel'
-        })}
-        position="left"
-      >
-        <EuiButtonIcon
-          className="mapLayerControl__openLayerTOCButton"
-          color="text"
-          onClick={openLayerTOC}
-          iconType="menuLeft"
-          aria-label={i18n.translate('xpack.maps.layerControl.openLayerTOCButtonAriaLabel', {
-            defaultMessage: 'Expand layers panel'
-          })}
-        />
-      </EuiToolTip>
-    );
-  }
-
+export function LayerControl({ isReadOnly, showAddLayerWizard, closeLayerTOC }) {
```

Comment:
Why was this remove? Looks like expand/collapse layer menu is now broken

Reply:
This one really threw me on first glance. It looks like it snuck in when I reverted the commit to separate the import workflow from the add layer workflow. That was a big and somewhat challenging revert. Sorry about this, I'll set it back to where it was.